src/client/src/Stream.c

```diff
@@ -751,6 +783,9 @@ STATUS putFrame(PKinesisVideoStream pKinesisVideoStream, PFrame pFrame)
 
     fixupFrame(pFrame);
 
+    // Set the last PutFrame time to current time
```

Comment:
Use this 

currentTime = pKinesisVideoClient->clientCallbacks.getCurrentTimeFn(pKinesisVideoClient->clientCallbacks.customData);

pKinesisVideoStream->lastPutFrameTimestamp = currentTIme;


The API can be expensive so we attempt to call it and store.

Reply:
What is the point to use getCurrentTimeFn instead of GETTIME().  I can make this change but would like the understand the benefit of this vs just calling GETTIME()?

Reply:
GETTIME() maps directly to the common defs. PIC takes the platform callbacks which includes the timer. We set this in the tests, etc. Retroactively, some of these have never really found any usage in the wild and we will be removing platform callbacks from C producer unless we get more reasons to keep them. The tests could directly override the function entry points in the common defs. But, for now, let's use consistency so we can easily migrate if we do